gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy

```diff
@@ -125,6 +125,10 @@ class AkkaGrpcPlugin implements Plugin<Project> {
         }
 
         project.afterEvaluate { Project p ->
+            if (akkaGrpcExt.scala) {
+                p.compileJava.enabled = false
```

Comment:
Aaah interesting. I guess this would interfere with hybrid java/scala projects - but I'm not sure how well-supported those are in Gradle in the first place?

Reply:
Yeah, you're right ! Probably I had to do a better, real fix and not that stupid as above

Reply:
Well, the possibility of having mixed scala \ java project also have to be handled in plugin itself.
Since now there're different options are set to the protoc compiler.
What do you think ?

Reply:
I think for maven/scala you're allowed to mix java and scala, but you can only generate either java or scala gRPC infrastructure. Would that be possible for gradle as well?

Reply:
Well, not sure what do you mean. It's 100% one can setup mixed java \ scala project in Gradle and classes will be properly compiled.

Reply:
Right - but with this change such a project would no longer compile the java classes, right? Or am I misunderstanding that?

Perhaps the condition could be changed so compileJava is only disabled when there are no java files?

Reply:
I am working on the fix right now.
But I asked smeth completely different  aboout mixed projects.